test/e2e/predictor/test_onnx.py

```diff
@@ -28,24 +27,16 @@
 from ..common.utils import predict
 from ..common.utils import KFSERVING_TEST_NAMESPACE
 
-from PIL import Image
-
 api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
 KFServing = KFServingClient(config_file="~/.kube/config")
-ONNX_EXAMPLE_IMAGE_FILE = './data/onnx_input.jpg'
-ONNX_EXAMPLE_JSON_FILE = './data/onnx_input.json'
-
-def prepare_onnx_example(path):
-    im = Image.open(path)
-    return np.array(im.resize((300, 400))).astype('float32') / 255
 
 def test_onnx_kfserving():
     service_name = 'isvc-onnx'
     default_endpoint_spec = V1alpha2EndpointSpec(
         predictor=V1alpha2PredictorSpec(
             min_replicas=1,
             onnx=V1alpha2ONNXSpec(
-                storage_uri='https://github.com/onnx/models/blob/master/vision/style_transfer/fast_neural_style/model/mosaic-9.onnx?raw=true',
+                storage_uri='https://github.com/tduffy000/kfserving-uri-examples/blob/master/onnx/frozen/iris.onnx?raw=true',
```

Comment:
perhaps we can put this object in `test/e2e/frozen_models` in order to not have this external dependency?

Reply:
if it's ok, I might address this in a small follow-up PR otherwise we'll end up with chicken and egg problem because it would have to be merged to be on the repo's master url